code/modules/mining/equipment/kinetic_crusher/_trophy.dm

```diff
@@ -0,0 +1,63 @@
+/**
+ * Base type for a crusher trophy. Does nothing except lie to you about causing errors.
+ */
+/obj/item/crusher_trophy
+	name = "tail spike"
+	desc = "A strange spike with no usage."
+	icon = 'icons/obj/mining_zones/artefacts.dmi'
+	icon_state = "tail_spike"
+	///Generic bonus to X var; if the trophy has a bonus effect, this is how much that effect is
+	var/bonus_value = 10
```

Comment:
i stand by having bonus values as vars because that makes adminbus funnier + who knows maybe i will actually do something about trophy upgrading in the future (after lavaland is basicmob'd approximately)

Reply:
I don't think a generic ability to upgrade trophies would make any sense because these values are treated as probabilities and seconds and "other" and aren't even utilised by all implementations. You can't add or multiply the value because this would have a wildly different effect on different items.

If you want these to be available for admin abuse then add them as editable vars on the individual item, allowing you to give them a descriptive name for what the admin is actually editing.